src/miniwiki.rs

```diff
@@ -58,9 +58,15 @@ fn render_class(class: &DumpClass) -> HtmlContent {
         .map(String::as_str)
         .unwrap_or(DEFAULT_DESCRIPTION);
 
+    let mut element_class: String = "dump-class".to_owned();

```

Comment:
Shouldn't need the type annotation here, there's only one way a string literal can turn into something owned.